src/taco-cli/cli/commands.json

```diff
@@ -399,6 +399,9 @@
             ,
             {
                 "name": "--target=TARGET", "description": "[CommandRunTargetDescription]"
+            },
+            {
+                "name":  "--list", "description": "[CommandRunListDescription]"
```

Comment:
we should definitely list that as an option to the command/synopsis. Description, not so sure because it will make the string very long...